package.json

```diff
@@ -65,6 +65,7 @@
     "lodash.range": "3.0.1",
     "minilog": "2.0.8",
     "node-sass": "3.3.3",
+    "nyc": "8.3.2",
```

Comment:
NYC is already dependency of `node-tap`: https://github.com/tapjs/node-tap/blob/master/package.json

Any reason to include it here as well?